apps/fishing-map/features/workspace/shared/DatasetSourceField.tsx

```diff
@@ -37,9 +37,10 @@ function DatasetFilterSource({ dataview, hideColor }: DatasetFilterSourceProps)
   }
 
   return (
-    <div className={styles.filter} data-test="activity-filters-tags">
+    <div className={styles.filter} data-test="source-tags">
       <label>{t('layer.source', 'Sources')}</label>
       <TagList
+        testId="source-tag-item"
```

Comment:
Why did we changed these `data-test` attributes? This change is going to make the vms tests fail: https://github.com/GlobalFishingWatch/frontend/blob/ee8ab566b5e3141d4396bb88bc20becef285486a/apps/fishing-map-e2e/src/e2e/fishing-activity/vms.cy.ts#L10-L21